db/data/test-data/index.js

```diff
@@ -1,4 +1,6 @@
-exports.articleData = require('./articles.js');
-exports.commentData = require('./comments.js');
-exports.topicData = require('./topics.js');
-exports.userData = require('./users.js');
+articleData = require('./articles.js');
+commentData = require('./comments.js');
+topicData = require('./topics.js');
+userData = require('./users.js');
+
+modules.export = { topicData, articleData, userData, commentData };
```

Comment:
What is your reasoning for changing the exports and is this working as expected?

Reply:
I just assumed that's what I had to do when task 1 said: 

"Make sure the index file exports an object with values of the data from that folder with the keys:
topicData
articleData
userData
commentData"

I had a misplaced s, but fixed it at some point after this pull request was sent.